app/src/main/res/values/colors.xml

```diff
@@ -131,37 +131,119 @@
     <color name="fx_mobile_private_layer_color_2" tools:ignore="UnusedResources">@color/photonInk50</color>
     <!-- Search -->
     <color name="fx_mobile_private_layer_color_3">@color/photonInk50</color>
+    <!-- Homescreen background, Toolbar -->
+    <color name="fx_mobile_private_layer_color_4_start" tools:ignore="UnusedResources">@color/photonPurple70</color>
```

Comment:
are the start/center/end here used for a gradient of some sort?

Reply:
Ya, it's a gradient